back-end/models/Token.js

```diff
@@ -14,7 +14,7 @@ const tokenSchema = new Schema(
     expireAt: {
       type: Date,
       default: Date.now,
-      expires: 36000,
+      expires: 360,
```

Comment:
Expiring after 6 minute? I think we can increase to 1 hour

Reply:
I thought link should expire about 5 minutes.Ok, I will expire it to 1 hour